src/widgets/component-new-edit/components/response-format/simple/simple-numeric.jsx

```diff
@@ -44,14 +58,42 @@ function ResponseFormatDatatypeNumeric({ name, required, readOnly }) {
           label={Dictionary.decimals}
           disabled={readOnly}
         />
-        <SelectMetaDataContainer
-          type="units"
-          name="unit"
-          label={Dictionary.unit}
-          emptyValue={Dictionary.unitEmptySelect}
-          mapMetadataFunction={mapUnitData}
+        <Field
+          name="isDynamicUnit"
+          component={ListRadios}
+          label={Dictionary.dynamicUnit}
           disabled={readOnly}
-        />
+          onChange={handleDynamicUnitChange}
+          required
+          // Convert string "true"/"false" to boolean true/false when storing in Redux form
+          parse={value => value === 'true'}
+          // Convert boolean true/false to string "true"/"false" when displaying the form
+          format={value => (value === true ? 'true' : 'false')}
```

Comment:
To simplify we could have defined this function as:
```
  value => `${value}`
```
But to each their own

Reply:
i'll change the comment, actually value can be undefined and i want to handle it as a false value